static/riotjs/wizard/participants_table.tag

```diff
@@ -65,10 +65,18 @@
 
     <div style="">
         <span>
```

Comment:
Reformatted these buttons to try to make them more clear.

Reply:
👍